src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/service/ReportService.kt

```diff
@@ -216,6 +217,36 @@ class ReportService(
     }.getOrNull()
   }
 
+  @Transactional
+  fun addDescriptionAddendum(id: UUID, descriptionAddendum: DescriptionAddendum): ReportWithDetails? {
+    return reportRepository.findById(id).getOrNull()?.let { reportEntity ->
```

Comment:
Actually, I think it may be better to use the eager version of this method given we're returning a `ReportWithDetails` (with all the various related objected as well)

Reply:
done here: https://github.com/ministryofjustice/hmpps-incident-reporting-api/pull/314/commits/5ecb59b4f911efc5dc8f280f906f5732224aaea4